backend/src/main/resources/Data.ts

```diff
@@ -114,7 +114,8 @@ const promotion1 = new Promotion(
   It’s never been easier to give back - all you have to do is eat your favourite fries (120-560 cals)! You can order them in-restaurant, at the drive thru, through McDelivery or you can order ahead on the McDonald’s app.
   If you’re looking for more ways to help families with sick children you can always round up your order - even when Fries for Good is over. When you finish placing your order, simply ask to “Round Up for RMHC” at participating McDonald’s restaurants, and your order will be rounded to the nearest dollar. The difference will be donated to RMHC Canada.`,
   new Date(),
-  new Date()
+  new Date(),
+  'promo1'
```

Comment:
Hehe sorry do you mind filling these in with restaurant names,  cause right now the sample data tries its best to use real descriptions/names